man/systemd-sysext.xml

```diff
@@ -0,0 +1,233 @@
+
+    <para>Another use case for the system extension concept is temporarily overriding OS supplied resources
+    with newer ones, for example to install a locally compiled development version of some low-level
+    component over the immutable OS image without doing a full OS rebuild or modifying the nominally
+    immutable image. (e.g. "install" a locally built package with <command>DESTDIR=/var/lib/extensions/mytest
+    make install &amp;&amp; systemd-osext --refresh</command>, making it available in
+    <filename>/usr/</filename> as if it was installed in the OS image itself.) This case works regardless if
+    the underlying host <filename>/usr/</filename> is managed as immutable disk image or is a traditional
+    package manager controlled (i.e. writable) tree.</para>
+  </refsect1>
+
+  <refsect1>
+    <title>Commands</title>
+
+    <para>The following command switches are understood:</para>
+
+    <variablelist>
+      <varlistentry>
+        <term><option>--merge</option></term>
+        <term><option>-m</option></term>
+        <listitem><para>Merges all currently installed system extension images into
+        <filename>/usr/</filename> and <filename>/opt/</filename>, by overmounting these hierarchies with an
+        <literal>overlayfs</literal> file system combining the underlying hierarchies with those included in
+        the extension images. This command will fail if the hierarchies area already
+        merged.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><option>--unmerge</option></term>
+        <term><option>-u</option></term>
+        <listitem><para>Unmerges all currently installed system extension images from
+        <filename>/usr/</filename> and <filename>/opt/</filename>, by unmounting the
+        <literal>overlayfs</literal> file systems created by <option>--merge</option>
+        prior.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><option>--refresh</option></term>
+        <term><option>-R</option></term>
+        <listitem><para>A combination of <option>--unmerge</option> and <option>--merge</option> in one: if
+        already mounted the existing <literal>overlayfs</literal> instance is unmounted temporarily, and then
+        replaced by a new version. This command is useful after installing/removing system extension images,
+        in order to update the <literal>overlayfs</literal> file system accordingly. If no system extensions
+        are installed when this command is executed, the equivalent of <option>--unmerge</option> is
+        executed, without establishing any new <literal>overlayfs</literal> instance. Note that currently
+        there's a brief moment where neither the old nor the new <literal>overlayfs</literal> file system is
+        mounted, and when thus all resources supplied by a system extension that exists continuously still
+        briefly disappears.</para></listitem>
```

Comment:
This reads super weird. Maybe `which implies that all resources supplied by a system extension that exists continuously still briefly dissapear.`?

Reply:
I'd break it up, as the sentence is already quite long.
```suggestion
        mounted. This implies that all resources supplied by a system extension will still
        briefly disappears even if it exists continuously during the refresh operation.</para></listitem>
```